dll/win32/kernel32/kernel32.spec

```diff
@@ -478,8 +478,8 @@
 @ stdcall GetFileSizeEx(long ptr)
 @ stdcall GetFileTime(long ptr ptr ptr)
 @ stdcall GetFileType(long)
-@ stub -version=0x600+ GetFinalPathNameByHandleA
-@ stub -version=0x600+ GetFinalPathNameByHandleW
+@ stdcall -version=0x600+ GetFinalPathNameByHandleA(long ptr long long)
```

Comment:
First parameter should be `ptr`. Unlike Wine, we consider handles pointers.

Reply:
ok

Reply:
I also mentionned the code is from wine in the source